guides/cookbook.md

```diff
@@ -1297,6 +1667,42 @@ writer.close();
 ```
 </div>
 
+<div class="tabpane Python" markdown="1">
+Regardless of the writer API used, the same CSV conversion method can be used.
```

Comment:
nit: make this present tense. 

"Regardless of the writer API in use, we can use the same CSV conversion method"

Reply:
Changed the wording.